libs/blocks/global-navigation/blocks/profile/profile.css

```diff
@@ -0,0 +1,133 @@
+:root {
+  --border-color: #e1e1e1;
+}
+
+.gnav-profile .gnav-profile-menu {
+  display: none;
+  position: fixed;
+  right: 0;
+  top: 56px;
+  min-width: 320px;
+  background: var(--color-white);
+  border: 1px solid var(--border-color);
+  border-radius: 0 0 4px 4px;
+  overflow: hidden;
+}
+
+.gnav-profile.is-open .gnav-profile-menu {
+  display: unset;
+}
+
+.gnav-profile-header {
+  display: flex;
+  padding: 20px;
+  align-items: start;
+}
+
+.gnav-profile-header .gnav-profile-img {
```

Comment:
Is `.gnav-profile-img` used anywhere else? Is there a reason for nesting it in `.gnav-profile-header`? Are there any specificity issues?

Reply:
Yeah, there is a feds-profile-img displayed on the nav and one within the dropdown - we just clone the existing image element, so we can either create it's own element or do it this way